src/index.html

```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+      autocomplete="off"
+    >
+      <fieldset>
+        <legend>Personal Information</legend>
+        <label class ="formField" for="surname">Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            required
+          >
+        </label>
+        <label class ="formField" for="name">Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            minlength="3"
+          >
+        </label>
+        <label class ="formField" for="age">How old are you?
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            max="100"
+            min="9"
+          >
+        </label>
+        <label class ="formField" for="dateOfBirth">Full date of birth
+          <input
```

Comment:
Если закрывающий тег на новой строке, то переноси текст
```suggestion
        </label>
        <label class ="formField" for="dateOfBirth">
          Full date of birth
          <input
```

Reply:
Добавил все с новой строки